builtin/providers/bitbucket/client.go

```diff
@@ -0,0 +1,65 @@
+package bitbucket
```

Comment:
What was the rationale to create a Bitbucket client in Terraform vs using or creating an API library?

I understand it is simple enough to take this approach but now a Bitbucket update of the API involves more projects on the internet manually updating their code vs pulling in a new version of the library.